www/js/app.js

```diff
@@ -154,7 +154,7 @@ define(['jquery', 'zimArchiveLoader', 'uiUtil', 'settingsStore','abstractFilesys
     document.getElementById('hideActiveContentWarningCheck').checked = params.hideActiveContentWarning;
     document.getElementById('showUIAnimationsCheck').checked = params.showUIAnimations;
     document.getElementById('titleSearchRange').value = params.maxSearchResultsSize;
-    document.getElementById('titleSearchRangeVal').innerHTML = encodeURIComponent(params.maxSearchResultsSize);
+    document.getElementById('titleSearchRangeVal').textContent = encodeURIComponent(params.maxSearchResultsSize);
```

Comment:
I suppose it's now unnecessary to call `encodeURIComponent` here (we probably did that to avoid the same warning)

Reply:
OK, will check.